package.json

```diff
@@ -145,16 +128,8 @@
       "@semantic-release/github"
     ]
   },
-  "resolutions": {
-    "node-sass": "4.7.2"
-  },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "tag": "latest"
-  },
-  "husky": {
```

Comment:
Why is husky removed?

Reply:
By accident from the snapshot PR, adding it back now.